app/views/jobs/index.html.haml

```diff
@@ -11,7 +11,7 @@
       Have an open position at your company? It is completely free to submit a job to our job board! Each job remains published for 60 days, and will be included in our newsletter while published. Please [submit your job here](/job_post_requests/new), and mind our [Job Board Policy](https://github.com/indyhackers/job_board_policy). Thank you!
 
 .job-list
-  - if @jobs.empty?
+  - if @jobs.blank?
```

Comment:
This is a minor optimization. Previously, this check and then listing the jobs resulted in two queries:

<details>
<summary>Previous log output</summary>

```
Started GET "/jobs" for 127.0.0.1 at 2020-08-23 20:56:06 -0400
Processing by JobsController#index as HTML
  Rendering jobs/index.html.haml within layouts/application
  Job Exists (0.5ms)  SELECT  1 AS one FROM "jobs" WHERE (jobs.published_at IS NOT NULL AND jobs.published_at <= '2020-08-24 00:56:06.308913') AND (jobs.published_at >= '2020-06-25 00:56:06.309622') LIMIT $1  [["LIMIT", 1]]
  Job Load (0.4ms)  SELECT "jobs".* FROM "jobs" WHERE (jobs.published_at IS NOT NULL AND jobs.published_at <= '2020-08-24 00:56:06.308913') AND (jobs.published_at >= '2020-06-25 00:56:06.309622') ORDER BY created_at DESC
  Rendered jobs/index.html.haml within layouts/application (5.1ms)
Completed 200 OK in 67ms (Views: 63.9ms | ActiveRecord: 0.9ms)
```
</details>

By calling `blank?` we force the collection to load in that check knowing we would be retrieving them all to list them anyway. This results in only one query:

<details>
<summary>Log output after the changes</summary>

```
Started GET "/jobs" for 127.0.0.1 at 2020-08-23 20:59:05 -0400
Processing by JobsController#index as HTML
  Rendering jobs/index.html.haml within layouts/application
  Job Load (0.6ms)  SELECT "jobs".* FROM "jobs" WHERE (jobs.published_at IS NOT NULL AND jobs.published_at <= '2020-08-24 00:59:05.193477') AND (jobs.published_at >= '2020-06-25 00:59:05.193984') ORDER BY created_at DESC
  Rendered jobs/index.html.haml within layouts/application (4.4ms)
Completed 200 OK in 59ms (Views: 36.8ms | ActiveRecord: 0.6ms)
```
</details>